### ISSUE TITLE
Adjust Penn State Branding

### ISSUE THREAD

**Issue by mbakeryo** [mlibrary/heliotrope#592]
Per Penn State's response:

- [x] 1) Can this band (the header) get wider and the PSUP and search box move to the right to look more like the screenshot of our website.
- [x] 2) The blue (footer band) is not matching the other two dark blues (header and Fulcrum links block in footer), but we'd like to see it in dark gray if possible.
- [x] 3) We'd rather not repeat the logo here (in the footer). Perhaps instead put the copyright line here and put our web address in the space where the copyright line currently is?
- [x] 3a) can you change the type from serif to the sans serif
- [x] 4) Change landing page copy to: Penn State University Press publishes books and journals of the highest quality for a worldwide audience of scholars, students, and non-academic readers, with an emphasis on core fields of the humanities and social sciences. This page features supplemental content for select PSU Press titles. You can find the complete catalog of our publications and additional information about the Press on our website.

**Comment by jmcglone** [comment]
@mbakeryo 

Re: #3 -- right now the partial template for a press footer assumes 1) that the logo/link to the press is in the top part of the footer (gray background block) and that 2) only a copyright statement is in the bottom of the footer. If we decide we want to accommodate this request, we'd need to open another ticket to do some development work that would allow the text/content in both blocks to be whatever a press wants it to be. @sethaj What do you think? keep extending seeds.db until we create an admin dashboard with a CRUD interface for customizing a press theme?

Re: #2 -- I happy to change the bottom footer to dark gray (there are a lot of grays...), but it should be noted that the blue in the footer IS the same as the header. The Fulcrum links block in the footer are the "Fulcrum brand blue".

**Comment by jmcglone** [comment]
Questions for Jennifer

1) in the PSU Press website example you shared yesterday, are both the logo and text clickable links?
2) when you ask to the make the header band wider, do you mean to increase its height or to decrease the width of the margins in the header?
3) By web address instead of the footer do you mean literally http://psupress.org or text like "Penn State Universtiy Press" that links to the PSU homepage?

**Comment by jmcglone** [comment]
Blocked by #
